src/main/java/com/bringframework/factory/ConfigBeanProcessor.java

```diff
@@ -0,0 +1,66 @@
+package com.bringframework.factory;
+
+import static com.bringframework.util.BeanUtils.createInstance;
+
+import com.bringframework.registry.BeanDefinition;
+import com.bringframework.registry.BeanDefinitionRegistry;
+import com.bringframework.registry.ConfigBeanDefinition;
+import com.bringframework.resolver.DependencyResolver;
+import java.util.Arrays;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Implementation of {@link BeanProcessor} that is responsible for processing {@link BeanDefinition}s
```

Comment:
proably `that is responsible for processing`  `BeanConfigDefinition` ?

Reply:
changed to `ConfigBeanDefinition`